presto-pinot-toolkit/pom.xml

```diff
@@ -22,6 +22,60 @@
         <dependency>
             <groupId>com.facebook.presto.pinot</groupId>
             <artifactId>pinot-driver</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
Just curious about if there are any automated tests around ensuring what all do we need to exclude ? Like would anything fail if we exclude less or more ?

Reply:
The maven plugin `` will fail the build.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (default) on project presto-pinot-toolkit: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
```